src/main/java/uk/gov/hmcts/cmc/claimstore/services/document/DocumentManagementService.java

```diff
@@ -61,7 +62,7 @@ public DocumentManagementService(
         AuthTokenGenerator authTokenGenerator,
         UserService userService,
         AppInsights appInsights,
-        @Value("${document_management.userRoles}") String userRoles
+        @Value("${document_management.userRoles}") List<String> userRoles
```

Comment:
this is unnecessary. The previous code was working fine

Reply:
we are converting to list here and then again converting to string. Please remove this redundant code.